src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
```

Comment:
- [ ] Kindly use configureStore instead of createStore because createStore has been deprecated :thinking: Kindly make this change :+1:

Reply:
Implemented. Thank you.